Wrong field name in Fn type creatation error message using @Type

### Zig Version

0.11.0

### Steps to Reproduce and Observed Output

Code: 
```zig
test {
    _ = @Type(.{
        .Fn = .{
            .calling_convention = .Unspecified,
            .alignment = 1,
            .is_generic = false,
            .is_var_args = false,
            .return_type = i32,
            .params = &.{
                .{
                    .is_noalias = false,
                    .is_generic = false,
                    .type = null,
                },
            },
        },
    });
}
```
Output:
```shell
src\main.zig:6:9: error: Type.Fn.Param.arg_type must be non-null for @Type
    _ = @Type(.{
```

### Expected Output

The referenced field should be `type` not `arg_type`, like so
```shell
src\main.zig:6:9: error: Type.Fn.Param.type must be non-null for @Type
    _ = @Type(.{
```

--- a/src/Sema.zig
+++ b/src/Sema.zig
@@ -21071,7 +21071,7 @@ fn zirReify(
                 }
 
                 const param_type_val = opt_param_type_val.optionalValue(mod) orelse
-                    return sema.fail(block, src, "Type.Fn.Param.arg_type must be non-null for @Type", .{});
+                    return sema.fail(block, src, "Type.Fn.Param.type must be non-null for @Type", .{});
                 param_type.* = param_type_val.toIntern();
 
                 if (param_is_noalias_val.toBool()) {

